CHANGELOG.md

```diff
@@ -8,6 +8,28 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.1] - 2024-03-19
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.5.0...v1.5.1)
+
+### Added
```

Comment:
Please also mention:

- support for modular models https://github.com/openfga/openfga/pull/1443. cc @rhamzeh 
- support for throttling in Check https://github.com/openfga/openfga/pull/1440